app/Http/Controllers/StandController.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Exceptions\Stand\StandAlreadyAssignedException;
+use App\Exceptions\Stand\StandNotFoundException;
+use App\Models\Stand\Stand;
+use App\Rules\VatsimCallsign;
+use App\Services\NetworkDataService;
+use App\Services\StandService;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+
+class StandController extends BaseController
+{
+    /**
+     * @var StandService
+     */
+    private $standService;
+
+    public function __construct(StandService $standService)
+    {
+        $this->standService = $standService;
+    }
+
+    public function getStandsDependency(): JsonResponse
+    {
+        return response()->json($this->standService->getStandsDependency());
+    }
+
+    public function getStandAssignments(): JsonResponse
+    {
+        return response()->json($this->standService->getStandAssignments());
+    }
+
+    public function createStandAssignment(Request $request): JsonResponse
+    {
+        $invalidRequest = $this->checkForSuppliedData(
```

Comment:
Where is this method? I can't find it in the base or in the controller, or is it globally bound?

Reply:
Maybe, it is worth putting this in a form request object?

Reply:
It sits on `app\Http\Controllers\BaseController.php`